aspnetcore/fundamentals/configuration.md

```diff
@@ -42,23 +42,23 @@ The app reads and displays the following configuration settings:
 
 [!code-json[Main](configuration/sample/src/ConfigJson/appsettings.json)]
 
-Configuration consists of a hierarchical list of name-value pairs in which the nodes are separated by a colon. To retrieve a value, you access the `Configuration` indexer with the corresponding item’s key:
+Configuration consists of a hierarchical list of name-value pairs in which the nodes are separated by a colon. To retrieve a value, access the `Configuration` indexer with the corresponding item’s key:
 
 ```csharp
 Console.WriteLine($"option1 = {Configuration["subsection:suboption1"]}");
 ```
 
-To work with arrays in JSON-formatted configuration sources, use a zero-based array index as part of the colon-separated string used to retrieve the value. For instance, to get the name of the first item in the `wizards` array shown above, use the following code:
+Use an array index with a colon-separated string to access arrays in a JSON-formatted configuration. The following example gets the name of the first item in the preceding `wizards` array:
 
 ```csharp
 Console.Write($"{Configuration["wizards:0:Name"]}, ");
 ```
 
 Name/value pairs written to the built in `Configuration` providers are **not** persisted, however, you can create a custom provider that saves values. See [custom configuration provider](xref:fundamentals/configuration#custom-config-providers).
 
-The sample above uses the configuration indexer to read values. When the `Configuration` object is available, the indexer is a convenient way to access setting. To flow information from configuration into other parts of the app outside of `Startup`, we recommend using the [options pattern](xref:fundamentals/configuration#options-config-objects). We'll show that later in this document.
+The preceding sample uses the configuration indexer to read values. To access configuration outside of `Startup`, use the [options pattern](xref:fundamentals/configuration#options-config-objects). The *options pattern* is shown later in this article.
```

Comment:
I thought we weren't supposed to use italic with links - that's double formatting.